<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layer_tiles_page</class>
 <widget class="QWidget" name="layer_tiles_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiles Layer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/ui/icons/Crystal_folder19.png</normaloff>:/ui/icons/Crystal_folder19.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_QTV_3">
     <property name="text">
      <string>messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_QTV_messages"/>
   </item>
   <item>
    <widget class="QLabel" name="label_qtv">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_addressUrl">
       <property name="toolTip">
        <string>such like &quot;http://localhost/osm_tiles2/%1/%2/%3.png&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_QTV_connect">
       <property name="text">
        <string>connect</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/ui/icons/cn11.png</normaloff>
         <normalon>:/ui/icons/cn6.png</normalon>:/ui/icons/cn11.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_QTV_4">
       <property name="text">
        <string>Cahce Expire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_QTV_cacheExpireDays">
       <property name="maximum">
        <number>365</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_QTV_5">
       <property name="text">
        <string>Days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_QTV_6">
       <property name="text">
        <string>Aplha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_QTV_Alpha">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_QTV_2">
       <property name="text">
        <string>Cahce Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_QTV_cacheFolder"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_QTV_browser">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_QTV_apply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/ui/icons/coffee.png</normaloff>:/ui/icons/coffee.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
